druid/examples/multiwin.rs

```diff
@@ -81,8 +82,16 @@ fn ui_builder() -> impl Widget<State> {
     col
 }
 
-fn make_delegate() -> AppDelegate<State> {
-    AppDelegate::new().event_handler(|event, data, _env, ctx| {
+struct MultiwinDelegate {}
```

Comment:
For a struct with no fields I prefer the 'unit struct' syntax, e.g. you can just write,

```rust
struct Delegate;
```